### ISSUE TITLE
Are we able to report Phishing Polkadot addresses?

### ISSUE THREAD

**Issue by jackesky** [polkadot-js/phishing#59]
I think i may have answered my own question being a, no.

**Comment by jacogr** [comment]
Yes. The address.json  file has them.

**Comment by jackesky** [comment]
thanks for your reply. I have 3 more questions.

1. Once the Phishing addresses are entered and updated into the Git hub database,  what happens to the address? Are future victims then unable to send funds into the scammers wallet as the address becomes listed as Phishing? 

2. Does that address also become useless in that the owner is unable to move crypto out of this Phishing address?

3. Does the URL link also become listed as Phishing on the Brave Browser, just like it is on Google Crome and Firefox web browser?

**Comment by jacogr** [comment]
To you questions -

For the addresses/sites to be used/detected it needs to be integrated into wallets. So on the polkadot-js extension (if installed) if you try for instance to go to https://polkadot.express, it does the following -

![image](https://user-images.githubusercontent.com/1424473/105629624-5adaae00-5e44-11eb-8ea6-34c4c6ddeea1.png)

(The above would be the same for any wallets forked off the polkadot-js extension, of which there are a couple)

In the same vein wallets need integration to check the actual addresses. At this point (since it is new), as far as I know the apps UI is the only one that checks against this list. For instance if you try to send to the polkadot.company address listed, you will be greeted with -

![image](https://user-images.githubusercontent.com/1424473/105629683-d2104200-5e44-11eb-9089-43a308bd4c70.png)

(At this point it doesn't disallow the account completely in the UI, but does protect the victims - originally wanted to go very hard, but rather decided to phase it in)

TL;DR The lists are available to be integrated as-is and it comes with a library for all checks for wallet providers.

**Comment by jacogr** [comment]
For now and future reference - if you have a favorite wallet, direct them to https://github.com/polkadot-js/phishing/tree/master/packages/phishing for integration.
